PLUGINS.md

```diff
@@ -154,6 +155,18 @@ A plugin to analyze and inspect Prow CI job results
 
 See [plugins/prow-job/README.md](plugins/prow-job/README.md) for detailed documentation.
 
+### Release Plugin
+
+A plugin to manage release configurations
```

Comment:
Both "release" plugin and "release configurations" could use some context-setting to better specify what releases and what configuration is meant here.